From c27c0cd96961e133ae3ee49193c4dc365238322b Mon Sep 17 00:00:00 2001
From: tstegema <tstegema@2eb42a87-3827-0410-b767-dc15055982e4>
Date: Fri, 6 Oct 2017 11:27:37 +0000
Subject: [PATCH 283/462] MANTIS 0047681 initial: CR "ID-Check before iDrop":
 implement PLF-Plugin checking boardingpass against ID-Check-server

added
  JarMORPlugin-HamIdCheckIDrop
  JarIDropIdCheck


git-svn-id: svn://localhost/SelfServiceBuild/trunk@452 2eb42a87-3827-0410-b767-dc15055982e4
---
 SelfServiceBuild/Build/java/AntProperties.xml | 2 ++
 SelfServiceBuild/Build/java/build.xml         | 1 +
 2 files changed, 3 insertions(+)

diff --git a/SelfServiceBuild/Build/java/AntProperties.xml b/SelfServiceBuild/Build/java/AntProperties.xml
index 23faae1..2ba70dd 100644
--- a/SelfServiceBuild/Build/java/AntProperties.xml
+++ b/SelfServiceBuild/Build/java/AntProperties.xml
@@ -257,6 +257,8 @@
 <property name="JarLoggingClient"           value="${JarDir}/LoggingClient.jar"/>
 <property name="JarLoggingInterface"        value="${JarDir}/LoggingInterface.jar"/>
 <property name="JarMipsSoapTools"           value="${JarDir}/mips-soap-tools.jar"/>
+<property name="JarMORPlugin-HamIdCheckIDrop" value="${JarDir}/mips-morplugin-HamIdCheckIDrop.jar"/>
+<property name="JarIDropIdCheck"            value="${JarDir}/idcheck.jar"/>
 
 <!-- slf4j -->
 <property name="JarSlf4jApi"                value="${DIR_GLOBALS}/ThirdParty/slf4j/1.7.21/slf4j-api-1.7.21.jar"/>
diff --git a/SelfServiceBuild/Build/java/build.xml b/SelfServiceBuild/Build/java/build.xml
index 6804bcc..4dcadb1 100644
--- a/SelfServiceBuild/Build/java/build.xml
+++ b/SelfServiceBuild/Build/java/build.xml
@@ -35,6 +35,7 @@
       ${PlatformJavaDir}/PlatformManager,
       ${PlatformJavaDir}/Soap-Tools,
       ${PlatformJavaDir}/LoggingClient,
+      ${PlatformJavaDir}/MORPlugin-HamIdCheckIDrop,
       ${PlatformJavaDir}/ApplicationAdapter,
       ${PlatformJavaDir}/AdminApplication,
       ${PlatformJavaDir}/SSDMApplication,
-- 
2.41.0.windows.1

